[2023-07-19 16:59:52,108] 187 werkzeug -INFO - [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:5000
 * Running on http://192.168.1.43:5000
[2023-07-19 16:59:52,108] 187 werkzeug -INFO - [33mPress CTRL+C to quit[0m
[2023-07-19 17:01:26,773] 187 werkzeug -INFO - 127.0.0.1 - - [19/Jul/2023 17:01:26] "[33mGET /predict HTTP/1.1[0m" 404 -
[2023-07-19 17:01:26,849] 187 werkzeug -INFO - 127.0.0.1 - - [19/Jul/2023 17:01:26] "[33mGET /favicon.ico HTTP/1.1[0m" 404 -
[2023-07-19 17:01:37,765] 1414 app -ERROR - Exception on / [GET]
Traceback (most recent call last):
  File "D:\project\ml-project-2\venv\lib\site-packages\flask\app.py", line 2190, in wsgi_app
    response = self.full_dispatch_request()
  File "D:\project\ml-project-2\venv\lib\site-packages\flask\app.py", line 1486, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "D:\project\ml-project-2\venv\lib\site-packages\flask\app.py", line 1484, in full_dispatch_request
    rv = self.dispatch_request()
  File "D:\project\ml-project-2\venv\lib\site-packages\flask\app.py", line 1469, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
  File "app.py", line 16, in index
    return render_template('index.html')
  File "D:\project\ml-project-2\venv\lib\site-packages\flask\templating.py", line 150, in render_template
    template = app.jinja_env.get_or_select_template(template_name_or_list)
  File "D:\project\ml-project-2\venv\lib\site-packages\jinja2\environment.py", line 1081, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "D:\project\ml-project-2\venv\lib\site-packages\jinja2\environment.py", line 1010, in get_template
    return self._load_template(name, globals)
  File "D:\project\ml-project-2\venv\lib\site-packages\jinja2\environment.py", line 969, in _load_template
    template = self.loader.load(self, name, self.make_globals(globals))
  File "D:\project\ml-project-2\venv\lib\site-packages\jinja2\loaders.py", line 126, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "D:\project\ml-project-2\venv\lib\site-packages\flask\templating.py", line 64, in get_source
    return self._get_source_fast(environment, template)
  File "D:\project\ml-project-2\venv\lib\site-packages\flask\templating.py", line 98, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: index.html
[2023-07-19 17:01:37,773] 187 werkzeug -INFO - 127.0.0.1 - - [19/Jul/2023 17:01:37] "[35m[1mGET / HTTP/1.1[0m" 500 -
[2023-07-19 17:03:08,876] 187 werkzeug -INFO - 127.0.0.1 - - [19/Jul/2023 17:03:08] "[33mGET /predict HTTP/1.1[0m" 404 -
[2023-07-19 17:03:15,181] 1414 app -ERROR - Exception on / [GET]
Traceback (most recent call last):
  File "D:\project\ml-project-2\venv\lib\site-packages\flask\app.py", line 2190, in wsgi_app
    response = self.full_dispatch_request()
  File "D:\project\ml-project-2\venv\lib\site-packages\flask\app.py", line 1486, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "D:\project\ml-project-2\venv\lib\site-packages\flask\app.py", line 1484, in full_dispatch_request
    rv = self.dispatch_request()
  File "D:\project\ml-project-2\venv\lib\site-packages\flask\app.py", line 1469, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
  File "app.py", line 16, in index
    return render_template('index.html')
  File "D:\project\ml-project-2\venv\lib\site-packages\flask\templating.py", line 150, in render_template
    template = app.jinja_env.get_or_select_template(template_name_or_list)
  File "D:\project\ml-project-2\venv\lib\site-packages\jinja2\environment.py", line 1081, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "D:\project\ml-project-2\venv\lib\site-packages\jinja2\environment.py", line 1010, in get_template
    return self._load_template(name, globals)
  File "D:\project\ml-project-2\venv\lib\site-packages\jinja2\environment.py", line 969, in _load_template
    template = self.loader.load(self, name, self.make_globals(globals))
  File "D:\project\ml-project-2\venv\lib\site-packages\jinja2\loaders.py", line 126, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "D:\project\ml-project-2\venv\lib\site-packages\flask\templating.py", line 64, in get_source
    return self._get_source_fast(environment, template)
  File "D:\project\ml-project-2\venv\lib\site-packages\flask\templating.py", line 98, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: index.html
[2023-07-19 17:03:15,182] 187 werkzeug -INFO - 127.0.0.1 - - [19/Jul/2023 17:03:15] "[35m[1mGET / HTTP/1.1[0m" 500 -
[2023-07-19 17:03:51,678] 1414 app -ERROR - Exception on /predictdata [GET]
Traceback (most recent call last):
  File "D:\project\ml-project-2\venv\lib\site-packages\flask\app.py", line 2190, in wsgi_app
    response = self.full_dispatch_request()
  File "D:\project\ml-project-2\venv\lib\site-packages\flask\app.py", line 1486, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "D:\project\ml-project-2\venv\lib\site-packages\flask\app.py", line 1484, in full_dispatch_request
    rv = self.dispatch_request()
  File "D:\project\ml-project-2\venv\lib\site-packages\flask\app.py", line 1469, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
  File "app.py", line 21, in predict_datapoint
    return render_template('home.html')
  File "D:\project\ml-project-2\venv\lib\site-packages\flask\templating.py", line 150, in render_template
    template = app.jinja_env.get_or_select_template(template_name_or_list)
  File "D:\project\ml-project-2\venv\lib\site-packages\jinja2\environment.py", line 1081, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "D:\project\ml-project-2\venv\lib\site-packages\jinja2\environment.py", line 1010, in get_template
    return self._load_template(name, globals)
  File "D:\project\ml-project-2\venv\lib\site-packages\jinja2\environment.py", line 969, in _load_template
    template = self.loader.load(self, name, self.make_globals(globals))
  File "D:\project\ml-project-2\venv\lib\site-packages\jinja2\loaders.py", line 126, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "D:\project\ml-project-2\venv\lib\site-packages\flask\templating.py", line 64, in get_source
    return self._get_source_fast(environment, template)
  File "D:\project\ml-project-2\venv\lib\site-packages\flask\templating.py", line 98, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: home.html
[2023-07-19 17:03:51,679] 187 werkzeug -INFO - 127.0.0.1 - - [19/Jul/2023 17:03:51] "[35m[1mGET /predictdata HTTP/1.1[0m" 500 -
[2023-07-19 17:03:55,372] 1414 app -ERROR - Exception on /predictdata [GET]
Traceback (most recent call last):
  File "D:\project\ml-project-2\venv\lib\site-packages\flask\app.py", line 2190, in wsgi_app
    response = self.full_dispatch_request()
  File "D:\project\ml-project-2\venv\lib\site-packages\flask\app.py", line 1486, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "D:\project\ml-project-2\venv\lib\site-packages\flask\app.py", line 1484, in full_dispatch_request
    rv = self.dispatch_request()
  File "D:\project\ml-project-2\venv\lib\site-packages\flask\app.py", line 1469, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
  File "app.py", line 21, in predict_datapoint
    return render_template('home.html')
  File "D:\project\ml-project-2\venv\lib\site-packages\flask\templating.py", line 150, in render_template
    template = app.jinja_env.get_or_select_template(template_name_or_list)
  File "D:\project\ml-project-2\venv\lib\site-packages\jinja2\environment.py", line 1081, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "D:\project\ml-project-2\venv\lib\site-packages\jinja2\environment.py", line 1010, in get_template
    return self._load_template(name, globals)
  File "D:\project\ml-project-2\venv\lib\site-packages\jinja2\environment.py", line 969, in _load_template
    template = self.loader.load(self, name, self.make_globals(globals))
  File "D:\project\ml-project-2\venv\lib\site-packages\jinja2\loaders.py", line 126, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "D:\project\ml-project-2\venv\lib\site-packages\flask\templating.py", line 64, in get_source
    return self._get_source_fast(environment, template)
  File "D:\project\ml-project-2\venv\lib\site-packages\flask\templating.py", line 98, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: home.html
[2023-07-19 17:03:55,373] 187 werkzeug -INFO - 127.0.0.1 - - [19/Jul/2023 17:03:55] "[35m[1mGET /predictdata HTTP/1.1[0m" 500 -
[2023-07-19 17:04:32,574] 1414 app -ERROR - Exception on /predictdata [GET]
Traceback (most recent call last):
  File "D:\project\ml-project-2\venv\lib\site-packages\flask\app.py", line 2190, in wsgi_app
    response = self.full_dispatch_request()
  File "D:\project\ml-project-2\venv\lib\site-packages\flask\app.py", line 1486, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "D:\project\ml-project-2\venv\lib\site-packages\flask\app.py", line 1484, in full_dispatch_request
    rv = self.dispatch_request()
  File "D:\project\ml-project-2\venv\lib\site-packages\flask\app.py", line 1469, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
  File "app.py", line 21, in predict_datapoint
    return render_template('home.html')
  File "D:\project\ml-project-2\venv\lib\site-packages\flask\templating.py", line 150, in render_template
    template = app.jinja_env.get_or_select_template(template_name_or_list)
  File "D:\project\ml-project-2\venv\lib\site-packages\jinja2\environment.py", line 1081, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
  File "D:\project\ml-project-2\venv\lib\site-packages\jinja2\environment.py", line 1010, in get_template
    return self._load_template(name, globals)
  File "D:\project\ml-project-2\venv\lib\site-packages\jinja2\environment.py", line 969, in _load_template
    template = self.loader.load(self, name, self.make_globals(globals))
  File "D:\project\ml-project-2\venv\lib\site-packages\jinja2\loaders.py", line 126, in load
    source, filename, uptodate = self.get_source(environment, name)
  File "D:\project\ml-project-2\venv\lib\site-packages\flask\templating.py", line 64, in get_source
    return self._get_source_fast(environment, template)
  File "D:\project\ml-project-2\venv\lib\site-packages\flask\templating.py", line 98, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: home.html
[2023-07-19 17:04:32,575] 187 werkzeug -INFO - 127.0.0.1 - - [19/Jul/2023 17:04:32] "[35m[1mGET /predictdata HTTP/1.1[0m" 500 -
